syntax = "proto3";

import "genshin/packet/proto/Item.proto";
import "genshin/packet/proto/StoreType.proto";

message PlayerStoreNotify {
	repeated Item item_list = 3;
	uint32 weight_limit = 8;
	StoreType store_type = 10;
}